Library/Formula/libssh2.rb

```diff
@@ -45,4 +44,19 @@ def install
     system "./configure", *args
     system "make", "install"
   end
+
+  test do
+    (testpath/"test.c").write <<-EOS.undent
+     #include <libssh2.h>
+
+     int main(void)
+     {
+      libssh2_exit();
+      return 0;
+     }
+    EOS
```

Comment:
Thanks Tim! Will try to ensure I hit the two-step indent inside testfiles in future :).